dhall-docs/src/Dhall/Docs.hs

```diff
@@ -51,20 +69,114 @@ parseOptions :: Parser Options
 parseOptions =
         Options
     <$> Options.Applicative.strOption
-        ( Options.Applicative.long "path"
-       <> Options.Applicative.metavar "PATH"
-       <> Options.Applicative.help "Root folder of your dhall package" )
+        ( Options.Applicative.long "input"
+       <> Options.Applicative.metavar "INPUT"
+       <> Options.Applicative.help "Directory of your dhall package" )
     <*> parseStrategy
+    <*> Options.Applicative.strOption
+        ( Options.Applicative.long "output"
+       <> Options.Applicative.metavar "OUTPUT"
+       <> Options.Applicative.help "Directory where your docs will be generated"
+       <> Options.Applicative.value "docs" )
 
 -- | `ParserInfo` for the `Options` type
 parserInfoOptions :: ParserInfo Options
 parserInfoOptions =
+    let progDesc = "Generate HTML documentation from a dhall package or file" in
     Options.Applicative.info
         (Options.Applicative.helper <*> parseOptions)
         (   Options.Applicative.fullDesc
-        <>  Options.Applicative.progDesc "Generate HTML documentation from a dhall package or file"
+        <>  Options.Applicative.progDesc progDesc
         )
 
+{-| Fetches a list of all dhall files in a directory. This is not the same
+    as finding all files that ends in @.dhall@, but finds all files that
+    successfully parses as a valid dhall file.
+
+    The reason it doesn't guide the search by its extension is because of the
+    dhall @Prelude@ (<https://github.com/dhall-lang/dhall-lang/tree/master/Prelude>)
```

Comment:
Haddocks support display text for links, which you can use like this:

```suggestion
    dhall <https://prelude.dhall-lang.org Prelude>
```

I also shortened the Prelude link